tests/ai/.gitignore

```diff
@@ -0,0 +1,5 @@
+teams*.yml
+env/
```

Comment:
also include `.env`?

Reply:
that's already included in root-level, but i'll also include it here